Source/Core/UICommon/DiscordPresence.h

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+// Adds support for Discord Rich Presence in to Dolphin
+
+#pragma once
+
+#include <string>
```

Comment:
It doesn't look like `std::string` is used in the header, so this can be removed.